bin/ch-checkns.c

```diff
@@ -74,9 +75,13 @@ void fatal_errno(int line)
 int main(int argc, char *argv[])
 {
 
+   if (argc == 1) {
+      fprintf(stderr, usage);
+      exit(EXIT_SUCCESS);
+   }
    if (argc >= 2 && strcmp(argv[1], "--help") == 0) {
       fprintf(stderr, usage);
-      return 0;
+      exit(EXIT_SUCCESS);
```

Comment:
It's not clear to me if there's any difference, but regardless, can you justify this change?